app/src/main/res/values/styles.xml

```diff
@@ -9,4 +9,14 @@
         <item name="android:windowTranslucentStatus" tools:targetApi="kitkat">true</item>
     </style>
 
-</resources>
\ No newline at end of file
+    <!-- Ваша AppTheme или другие темы и стили здесь -->
+
+    <!-- Кастомная AppTheme.Launcher. Устанавливает в качестве фона вашу drawable -->
+    <style name="AppTheme.Launcher">
+        <item name="android:windowBackground">@drawable/launch_screen</item>
```

Comment:
@SimonMaha for Splash screen there should be a separate Activity. For example, "SplashActivity". 
It should be defined as a launch activity in manifest and redirect a user to the next screen after a couple seconds. It can be done with Handler class